### PR TITLE
Updated key features section

### PR DESCRIPTION
**Pull request by evanxg852000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,17 +26,6 @@ Sub-second search & analytics engine on cloud storage
 </h4>
 <br/>
 
-<b>Quickwit 0.6 is around the corner and is set to redefine your search experience on multi-terabyte datasets with a host of powerful features!</b>
```
Comment (fmassot) [inline README.md:29]: There is a pending PR on the README, can you revert this change?

File: docs/overview/introduction.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ Check out our guides to see how you can use Quickwit:
 - OTEL-native for [logs](../log-management/overview.md) and [traces](../distributed-tracing/overview.md)
 - [Jaeger-native](../distributed-tracing/plug-quickwit-to-jaeger.md)
 - [RESTful API](../reference/rest-api.md)
+- [Elasticsearch API](../reference/es_compatible_api.md) with ingest and query DSL support.
```
Comment (fmassot) [inline docs/overview/introduction.md:43]: Can you take the same features as the ones that will be on the readme page? 

see https://github.com/quickwit-oss/quickwit/pull/3491/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47-R56

Comment (evanxg852000) [reply docs/overview/introduction.md:43]: Ok

**Review by fmassot — APPROVED** [review]
